.gitignore

```diff
@@ -46,6 +46,7 @@ npm-debug.log
 # File-system cruft and temporary files
 .DS_Store
 .idea
+.vscode
```

Comment:
Strictly speaking, yes. But we both know that a lot of people don't have them, so we should put them in our `.gitignore`.